packages/types/src/oauth.ts

```diff
@@ -244,4 +244,8 @@ export type AuthenticateWithRedirectParams = {
    * Full URL or path to navigate after the OAuth flow completes.
    */
   redirectUrlComplete: string;
+  /**
+   * Whether to continue (i.e. PATCH) an existing SignUp (if present) or create a new SignUp
+   */
+  continueSignUp?: boolean;
```

Comment:
:upside_down_face: Not sure about property naming and whether this property should be part of the public API. 

How about `shouldCreateNew`? Let's see what the others think.

Reply:
Since `shouldCreateNew` should be true by default, I'd prefer to go for a flag that should be false by default.

We have the concept of "continuing a sign-up" already, so it's not a new term.

Will proceed as is to unblock this fix, but open to revisiting this.